mender-shell Yocto updates: new variable and config functionality (#1264)
mendersoftware/mender-docs: 05.System-updates-Yocto-Project/05.Customize-Mender/docs.md

Alternatively, add the snippet to your `local.conf`.

Mender-shell provides several [configuration
options](../../03.Add-ons/01.Remote-Terminal/30.Mender-shell-configuration-file/50.Mender-shell-configuration-options/docs.md).
A configuration file with the required [`ServerURL`
field](../../03.Add-ons/01.Remote-Terminal/30.Mender-shell-configuration-file/50.Mender-shell-configuration-options/docs.md#serverurl),
set via [`MENDER_SERVER_URL`
variable](../99.Variables/docs.md#mender_server_url), and [`User`
field](../../03.Add-ons/01.Remote-Terminal/30.Mender-shell-configuration-file/50.Mender-shell-configuration-options/docs.md#user),
set via [`MENDER_SHELL_USER`

> variable](../99.Variables/docs.md#mender_shell_user). These shall be set by via

Reviewer: suggested change: 'These shall be set' -> '_You can set these_'
suggested change: 'by via' -> 'via' (or something else)

Author: > suggested change: 'These shall be set' -> 'You can set these'

@merlin-northern Unless you use your own `mender-shell.conf`, these are mandatory. Suggestion: "You have to set these"

Author: Okey, I decided for "you ca set". Technically speaking is voluntary, but the resulting default will be useless. But okey

> your own `.bbappend` recipe file, or via your `local.conf` file, for example:

Reviewer: I am not sure do we need comma here? "recipe file, or via your" -> "recipe file or via your"

```bash
MENDER_SERVER_URL = "https://hosted.mender.io"
MENDER_SHELL_USER = "root"
```

> To add optional fields, or override the values for the required ones, create your own `mender-shell.conf` and

Reviewer 2: ```suggestion
To add optional fields, or override the values for the required ones, create your own `mender-shell.conf` and
```

Author: I can't spot the difference...

Reviewer 2: Wtf, sry. I had to refresh whilst doing the review, so something happened obvs
```suggestion
Add optional, or override required fields, with your own `mender-shell.conf` file. Then add the file to the build with...
```
